justfile

```diff
@@ -11,7 +11,7 @@ fmt:
 
 #Will run the controller
 run:
-   go run service-mirror.go
+   go run *.go
```

Comment:
hate to be a stickler for this but can we rename `service-mirror.go` to `main.go`? We should probably set it up with a pkg (or internal better yet) directory too. Let's do it in a follow up don't wanna block on it.